### DISCUSSION TITLE
[Question]: Change detection

### DISCUSSION THREAD

**Discussion by jchretien** [blakeblackshear/frigate#12729] [Ask A Question]
### What is your question:

Let's say I sent a positive detection to Frigate+ but once in verification in Frigate+ I realize I was mistaken and want to change to "Not a person", how do I do it?

![image](https://github.com/user-attachments/assets/c385ffff-84ea-4514-ac09-0d73d0404e3f)

**Comment by NickM-27** [comment]
You can't, specific information is needed that isn't available so there is no easy way to undo it. Best to just delete the box, label all true objects  and submit the image.
